### PR TITLE
Added python integration test core modules

### PR DESCRIPTION
**Pull request by taishi8117**

### CODE REVIEW

File: integration-test/src/python/pyheron/integration_test/core/BUILD [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package(default_visibility = ["//visibility:public"])
+
+load("/tools/rules/pex_rules", "pex_library", "pex_binary")
+
+pex_library(
+  name = "pyheron_integ_core",
```
Comment (kramasamy) [inline integration-test/src/python/pyheron/integration_test/core/BUILD]: keep it a full name 'pyheron_integration_core'

**Comment by taishi8117** [comment]
Will remove pyheron dir, as it's unnecessary

**Comment by kramasamy** [comment]
👍

**Comment by kramasamy** [comment]
@objmagic - any comments?

**Comment by objmagic** [comment]
@kramasamy looks fine. merged.

**Pull request merged**